conn.go

```diff
@@ -23,6 +24,7 @@ import (
 	"unicode"
 
 	"github.com/lib/pq/oid"
+	"github.com/spaolacci/murmur3"
```

Comment:
Thats a good point, when I did this I was thinking I'd need to pull the name out of pg_prepared_statements on the server at some point, that clearly doesn't need to happen. Let me remove that.